student.rb

```diff
@@ -5,6 +5,7 @@ class Student < Person
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
     @classroom = classroom
+    classroom&.add_student(self)
```

Comment:
- [ ] It's expected to have a classroom method that serves as the 'belongs-to' side (a student belongs to a classroom) that can be used later. Kindly have a classroom method to this file and then you could remove lines 7 and 9 and maintain the class instead. 👍🏽

```ruby
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```